lib/msal-common/src/app/module/AuthorizationCodeModule.ts

```diff
@@ -237,7 +237,7 @@ export class AuthorizationCodeModule extends AuthModule {
                     refreshToken: cachedTokenItem.value.refreshToken,
                     expiresOn: new Date(expirationSec * 1000),
                     account: account,
-                    userRequestState: ""
+                    userRequestState: MsalMethod.SILENT
```

Comment:
Isn't this supposed to be set in request validation? That is coming in msal-node, not sure about the msal-browser.

Reply:
i will double check, you may be correct. will get back to this comment